src/NexusMods.App.UI/Pages/Downloads/DownloadColumns.cs

```diff
@@ -0,0 +1,139 @@
+using NexusMods.App.UI.Controls;
+
+namespace NexusMods.App.UI.Pages.Downloads;
+
+/*
+ * DOWNLOADS COLUMN DEFINITIONS OVERVIEW  
+ * ====================================
+ * 
+ * This file defines the column structure for the Downloads TreeDataGrid.
+ * Each column implements ICompositeColumnDefinition<T> and maps to specific components.
+ * 
+ * COLUMN STRUCTURE (5 columns total):
+ * 
+ * 1. NAME+ICON COLUMN:
+ *    - Uses SharedColumns.Name from Controls/TreeDataGrid/SharedColumns.cs
+ *    - Components: NameComponent + ImageComponent  
+ *    - Shows download/mod name with icon
+ * 
+ * 2. GAME COLUMN (defined here as DownloadColumns.Game):
+ *    - Maps to: DownloadComponents.GameComponent (from DownloadComponents.cs)
+ *    - Shows: Game name + game icon
+ *    - Sortable by game name alphabetically
+ * 
+ * 3. SIZE COLUMN (defined here as DownloadColumns.Size):  
+ *    - Maps to: DownloadComponents.SizeProgressComponent (from DownloadComponents.cs)
+ *    - Format: "15.6MB of 56MB" (downloaded of total)
+ *    - Sortable by total file size
+ * 
+ * 4. SPEED COLUMN (defined here as DownloadColumns.Speed):
+ *    - Maps to: DownloadComponents.SpeedComponent (from DownloadComponents.cs)  
+ *    - Shows: Transfer rate "5.2 MB/s" or "--"
+ *    - Sortable by current transfer rate
+ * 
+ * 5. STATUS COLUMN (defined here as DownloadColumns.Status):
+ *    - Maps to: DownloadComponents.StatusComponent (from DownloadComponents.cs)
+ *    - Contains: Progress bar + Pause/Resume + Cancel + Kebab menu
+ *    - Consolidated all download actions into single column
+ *    - Sortable by completion percentage (0% to 100%)
+ */
```

Comment:
Put the comments next to the actual columns definitions, otherwise they will become outdated too easily.

Reply:
I just added the Collections column to the Library.
Plan is to allow users to toggle which columns they want to see in the future (no idea how yet), but that means there will be opportunity to add a bunch of columns.

Reply:
Ah. My original intent at the time was to have a listing; since some columns were reused from library and other views.
However, in the end, I had to duplicate some of those columns instead in the final branch/PR, due to the usage of generics.

I can inline it as comments to the items in this case; yeah; that works.

I'll add that to my notes for the final PR to avoid conflicts.